.travis.yml

```diff
@@ -29,8 +29,10 @@ node_js:
 
 ## Use this to prepare your build for testing
 ## e.g. copy database configurations, environment variables, etc.
-# before_script:
-#
+## http://stackoverflow.com/questions/12336566/travis-ci-with-jasmine-node
+before_script:
```

Comment:
Can we move this to the ``install`` section instead? Since we might put the ``npm start`` here.